stdlib/VERSIONS

```diff
@@ -46,6 +46,7 @@ _pydecimal: 3.5-
 _random: 2.7-
 _sitebuiltins: 3.4-
 _socket: 3.0-  # present in 2.7 at runtime, but not in typeshed
+_sqlite3: 2.7-
```

Comment:
```suggestion
_sqlite3: 3.0-
```
We just merged a PR removing all the 2.7 lines.

Also, general note: I was away for a few weeks and I see you have a number of open PRs that haven't been merged yet. I'm just glancing through my notifications at the moment, but when I find some time I'll try to go over open nondraft typeshed PRs to review what I can.

Reply:
Appreciated, but no rush